BlackJackets API/src/main/java/BlackJackets/BlackJackets/Controllers/YouTubeController.java

```diff
@@ -0,0 +1,23 @@
+package BlackJackets.BlackJackets.Controllers;
+
+import BlackJackets.BlackJackets.service.YouTubeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/youtube")
```

Comment:
Needs `@CrossOrigin(origins = "http://localhost:5173")` for the api to be called.

Reply:
Done